.github/workflows/bazel_test.yml

```diff
@@ -0,0 +1,11 @@
+name: Bazel Test
+
+on: [push, pull_request]
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Dummy test
+        run: |
+          bazel test //...
```

Comment:
The test seems to fail because it cannot find the workspace. Do you know which `cwd` this is running in?

Reply:
As far as I can find online, the working-directory should be able to explicitly define the directory in which the command runs, but so far I failed with setting it as "." and "./"

Reply:
Does the WORKSPACE need a line like
workspace(name = "bazel-rules-fuzzing")?

Once I hit this issue, and i used print commands in the github workflow for debugging :P

Reply:
It seems adding that line to the WORKSPACE file doesn't work :(

Reply:
> Run echo `pwd`
/home/runner/work/bazel-rules-fuzzing/bazel-rules-fuzzing

I print the working directory path. It seems the Actions creates a sub-directory with the same name to run the command.

Reply:
> ERROR: The 'test' command is only supported from within a workspace (below a directory having a WORKSPACE file).


I wonder if this is happening because it's a top level test in the same directory as the WORKSPACE

Reply:
Also maybe it's worth printing whats in the directory. I looked at my example worfklow, and I had an action to checkout the repo. 
```
    steps:
    - name: Checkout repository
      uses: actions/checkout@v2
```

Reply:
I think the checkout does the magic. Now the bazel test can be successfully triggered.

Do you think I should revert recent commits to push a more clear update? I mean, cutting out those messy and debug commits.

Reply:
Flattening the commits upon PR merging sounds good to me. The Github UI should handle that.